src/main/java/com/example/demo/Controller/JavaQuestionController.java

```diff
@@ -0,0 +1,32 @@
+package com.example.demo.Controller;
+
+import com.example.demo.Services.JavaQuestionService;
+import com.example.demo.Services.QuestionService;
+import com.example.demo.model.Question;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+    private final JavaQuestionService questionService;
+    public JavaQuestionController(JavaQuestionService questionService) {
+        this.questionService = questionService;
+    }
+    @PostMapping("/add")
+    public Question addQuestion(@RequestParam String questionText,@RequestParam String questionAnswer) {
+        Question question = new Question(questionText, questionAnswer);
+        questionService.addQuestion(question);
+        return question;
+    }
+    @DeleteMapping("/remove")
+    public Question removeQuestion(@RequestParam String questionText, String questionAnswer) {
```

Comment:
аннотацию надо ставить перед каждым параметром
```suggestion
    public Question removeQuestion(@RequestParam String questionText, @RequestParam String questionAnswer) {
```